### ISSUE TITLE
Compiling qemu arm from sources - linker errors

### ISSUE THREAD

**Issue by Nodraak** [gnuarmeclipse/qemu#38]
Hi !

I am trying to compile the qemu sources but when I run `make` the build fails with multiple linker errors :

* undefined reference to `wav_start_capture'
* undefined reference to `omap_badwidth_read8'
* undefined reference to `AUD_write'
* and so on

How do I install the needed dependencies (I tried to find the libraries but could not find anything) ?

I am compiling qemu-gae-2.8.0-20161227 on Debian 8 (Jessie), without docker.

**Comment by ilg-ul** [comment]
https://github.com/gnuarmeclipse/build-scripts/blob/master/scripts/build-qemu.sh

**Comment by ilg-ul** [comment]
http://gnuarmeclipse.github.io/qemu/build-procedure/

**Comment by Nodraak** [comment]
Hi again. I finally found some time to work on that again. I can successfully build qemu and edit the source code. Thanks a lot !

I am trying to add new nucleo boards : L432KC (and maybe F303K8 and L476RG).
Is there any guide on what step to take to add a new board ? That would be handy.
I have tried to edit the files in `include/hw/cortexm/stm32` and `hw/cortexm/stm32` by copying another board but I am missing the svd files. How can I get/generate them ?
